### ISSUE TITLE
Mystical Agriculture seeds seem unaffected by Growth Accelerators

### ISSUE THREAD

**Issue by b12arr0** [ResqDiver1317/ThirdPeril_PerilousSkies#1]
The Mystical Agriculture seeds do not seem to grow any faster while using either Mystical Agriculture Growth Accelerator blocks or the Tiny Progressions Growth Crystals under them. 

**To Reproduce**
1. Plant a Mystical Agriculture seed (ie.  Inferium seed) on farmland dirt.
2. Place a growth accelerator or growth crystal below the farmland.
3. The growth accelerator or crystal appear to have no affect on the growth of the seed.  

**Expected behavior**
I would expect the MA Growth Accelerator blocks would have a noticable affect on speeding up MA seed growth.  

**Notes**
Using a Cyclic sprinkler does show some noticable growth affect on the seeds. 
I did test the Growth crystal under normal crops and it does noticably affect growth speed of normal crops(wheat, beets, etc).

**Comment by b12arr0** [comment]
After doing some testing all morning, I need to correct and close this issue.

All testing was done with a 9x9 area with a single water source in the middle, with 80 Tier 1 Mystical Agriculture Inferium seeds.

With a single Tiny Progressions Tier 3 Growth Crystal, it appears that there was no difference than with just the single water source.  This was noticed by myself and Dragon__Realms.

I disabled the harvester so I get accurate data.

With just a water source, It took 3 hours for 70 of the seeds to reach maturity, the 10 remaining were 0-85% mature.

I redid the test with just a single TP Tier 3 Growth Crystal, it took 11 minutes and 30 seconds to get  76 mature seeds, 4 with 57% or better maturity. 

With a sprinkler and water, it took 23 and half minutes to get to 79 mature seeds, and 1 seed at 85%

With water and 80 MA growth accelerators, it took 20 and a half minutes to get to 69 mature seeds, and 1 seed at 42%

With 3 Tiny Progressions Tier 3 Growth Crystals, 1 Tier 2 growth crystal, 1 Tier 1 growth crystal, it took 15 minutes and 30 seconds to get to 78 mature seeds, 1 seed at 85% and 1 seed at 57%

With 3 TP Tier 3 Growth Crystals, 1 Tier 2 Growth Crystal, 1 Tier 1 Growth Crystal, 1 water source, 1 sprinkler, and 75 growth accelerators, it took 12 minutes and 17 seconds to reach 100% maturity


So, in conclusion,  the MA accelerators and TP Growth Crystals, and sprinklers all work as expected and do increase growth times. I think the use of the harvester made it seem like they were not.   I'm not sure why having all the TP growth crystals, MA growth accelerators, and sprinkler did worse than the single Tier 3 Growth Crystal.   Very weird.
